packages/shared/src/components/fields/ImageInput.tsx

```diff
@@ -44,7 +44,7 @@ function ImageInput({
 }: ImageInputProps): ReactElement {
   const inputRef = useRef<HTMLInputElement>();
   const [error, setError] = useState('');
-  const [image, setImage] = useState(initialValue);
+  const [image, setImage] = useState(initialValue || fallbackImage);
```

Comment:
I don't think it should always fallback, because of the OIDC flow. That will then always use the fallback?

Not sure if it's still the case, but that user to be why this was static null.

Reply:
The edge case here, if `initialValue` is empty no error will be thrown and the image will be broken. It happened to me while trying out this feature. This is why I applied this fix. We don't always use fallback only when `initialValue` is empty

Reply:
Hmm ok, just to make sure this doesn't break the OIDC or email flow as it used to do.

Reply:
I'll double check